index.js

```diff
@@ -1 +1,120 @@
-console.log("Code your solution!")
+
+
+
+// console.log("Code your solution!")
+
+
+//1. query the form element
+const form = document.querySelector("#to-dos")
+//2. Add an event listener for submitting the form.
+form.addEventListener('submit', (event) => {
+    //3. prevent the page from refreshing
+    event.preventDefault();
+    console.log(event);
+
+    const list = event.target.list.value;
+    
+
+
+    addToList(list);
+
+    if (list.includes("\n")) {
+        const splitList = list.split("\n");
+        const sliceList = splitList.slice(1)
+        sliceList.forEach((each) => {
+            addToList(each)
+        })
+     } 
+
+
+    event.target.reset();
+
+})
+
+//4. get the text that's in each input box
+
+//5. give those values to my function 
+
+function listTemplate (thing) {
+// Check if there is no inputted value and return an error message
+    if (!thing) {
+        const msg = document.createElement("p");
+        msg.textContent = "Please enter something";
+        form.after(msg);
+        return msg
```

Comment:
Good job with the message. Something that would improve user experience is removing the p tag once the user creates a viable input. At the moment your message remain
![Screen Shot 2021-09-30 at 10 44 33 AM](https://user-images.githubusercontent.com/71154265/135477503-be3a551d-a783-49a2-8081-863eef48f749.png)
s.